src/CHANGES.md

```diff
@@ -1,4 +1,4 @@
-Changelog
+# Changelog
```

Comment:
interestingly, it seems like the rendered changelog page gets a "changelog" heading inserted automatically --> https://bids-specification.readthedocs.io/en/stable/CHANGES.html

previously we then had something like

---

# Changelog

Changelog

---

is it possible that this change will turn this into

---

# Changelog

# Changelog

---

?

Reply:
@sappelhoff Thanks for the feedback!

- I've reverted the change I'd made. So I believe that should have fixed it. I needed "Changelog" to be Heading 1 so that pandoc recognizes it to be a new section and doesn't concatenate with the previous section. However, I found a hack around it that didn't require the 'src/CHANGES.md' to be modified. 

- Most recent commit takes care of naming change and doesn't duplicate the pdf. I'm looking into how to make it part of the build and render it as part of the RTD

Reply:
> I've reverted the change I'd made. So I believe that should have fixed it. I needed "Changelog" to be Heading 1 so that pandoc recognizes it to be a new section and doesn't concatenate with the previous section. However, I found a hack around it that didn't require the 'src/CHANGES.md' to be modified.

actually ... perhaps it was good that you turned that "changelog" into a heading. Perhaps that would make it findable on our website: https://bids-specification.readthedocs.io/en/stable/CHANGES.html

when typing "changelog" into the search function, I don't get this site ... although the search function usually qorks quite well. :thinking: 

@franklin-feingold would it be a problem with our auto-changelog generator to change the first line in https://github.com/bids-standard/bids-specification/blob/master/src/CHANGES.md to be a markdown heading?

Reply:
if we change it the changelog generator regenerates (and commits) for each merged PR so that won't solve it - there may be something within the changelog generator that can be changed but nothing apparent

Reply:
So do I leave it as it is right now? or make the "changelog" a heading?

Reply:
We leave it as it is for now and look into whether we can tweak some changelog-generator settings to label the page with `# changelog` instead of `changelog`

Reply:
we'll see if https://github.com/bids-standard/bids-specification/pull/417 does the job.

Reply:
sounds good!

Reply:
@sappelhoff I spent some time looking up how this can be done 
> the pdf should be built concurrently with the READTHEDOCS build of the website, and then also hosted as a build artifact on readthedocs.
According to RTD's build process using `mkdocs`, simultaneous pdf generation isn't possible. https://docs.readthedocs.io/en/stable/builds.html
Perhaps, pdf can be created and then hosted on the built RTD version? However, I'm unsure how to fix this. Would appreciate any ideas/suggestions.

Reply:
I discussed with @franklin-feingold and @rwblair yesterday and we agreed that the most pragmatic way to go forward would be to split our PDF-keeping in two parts:

1. on each merged pull request to bids-specification, circle CI should build a pdf ... including the current version and date ... this pdf is then hosted in the "artifacts" tab of the circleCI build. --> we can generate a link that will always resolve to the most recently build pdf artifact on circleci. See this link as an example from the mne-bids repo: `https://circleci.com/api/v1.1/project/github/mne-tools/mne-bids/latest/artifacts/0/html/index.html?branch=master` ... it will always resolve to the **latest** index.html file built by circleci

2. whenever we do a **Release** of the bids specification, a maintainer builds the PDF and goes through the steps to host it permanently somewhere ... as currently discussed in https://github.com/bids-standard/bids-specification/pull/407

so for now, we simply need to figure out how to build the pdf using your script ... in circleci. Which is mainly a question of installing all needed packages there.

Reply:
this may come in very handy for getting latex to run on circleci: https://github.com/dante-ev/docker-texlive